static/js/containers/pages/checkout/CartPage.js

```diff
@@ -0,0 +1,276 @@
+        </div>
+
+        <div className="row order-summary-total">
+          <div className="col-12 px-3 py-3 py-md-0">
+            <div className="d-flex justify-content-between">
+              <div className="flex-grow-1">
+                <h5>Total</h5>
+              </div>
+              <div className="ml-auto">
+                <h5>${totalPrice}</h5>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        {SETTINGS.features.enable_discount_ui ? (
+          <div className="row">
+            <div className="col-12 mt-4 px-3 py-3 py-md-0">
+              Have a coupon?
+              <div className="d-flex justify-content-between flex-sm-column flex-md-row">
+                <input
+                  type="text"
+                  name="coupon-code"
+                  className="form-input flex-sm-grow-1"
+                />
+                <button className="btn btn-primary btn-red btn-halfsize mx-2 highlight font-weight-normal">
+                  Apply
+                </button>
+              </div>
+              <div className="text-primary mt-2 font-weight-bold cart-text-smaller">
+                Adding another coupon will replace the currently applied coupon.
+              </div>
+            </div>
+          </div>
+        ) : (
+          ""
+        )}
+
+        <div className="row">
+          <div className="col-12 text-center mt-4 mb-4">
+            <Button
+              type="link"
+              className="btn btn-primary btn-gradient-red highlight font-weight-bold text-white"
+              onClick={() => (window.location = "/checkout/to_payment")}
+            >
+              Place your order
+            </Button>
+          </div>
+        </div>
+
+        <div className="row">
+          <div className="col-12 px-3 py-3 py-md-0 cart-text-smaller">
+            By placing my order I agree to the Terms of Service, and Privacy
+            Policy.
+          </div>
+        </div>
+      </div>
+    )
+  }
+
+  render() {
+    const { cartItems, isLoading } = this.props
+
+    return (
+      <DocumentTitle
+        title={`${SETTINGS.site_name} | ${CART_DISPLAY_PAGE_TITLE}`}
+      >
+        <Loader isLoading={isLoading}>
+          <div className="std-page-body cart container">
+            <div className="row">
+              <div className="col-12 d-flex justify-content-between">
+                <h1 className="flex-grow-1">Checkout</h1>
+
+                <p className="text-right d-md-none">Help &amp; FAQs</p>
+              </div>
+            </div>
+
+            <div className="row d-none d-md-flex mb-3">
+              <div className="col-md-8">
+                <h4 className="font-weight-normal">
+                  You are about to purchase the following:
+                </h4>
+              </div>
+              <div className="col-md-4 text-right">
+                <h4 className="font-weight-normal">
+                  <a href="#">Help &amp; FAQs</a>
```

Comment:
```suggestion
                  <a href="https://mitxonline.zendesk.com/hc/en-us">Help &amp; FAQs</a>
```

Probably in a new tab/window too.